### DISCUSSION TITLE
Spotify-url-info help

### DISCUSSION THREAD

**Discussion by peginium** [Androz2091/discord-player#258] [General]
Hey im learning code and i have some problem im using discord player but i cant play spotify playlist but i can play spotify track is there any way to play playlist

**Comment by thaddeuskkr** [comment]
Yes, a way to use Spotify playlists were implemented a while ago, could you maybe attach an error message (if there is one)?

**Reply by peginium** [reply]
there isnt any error

**Reply by thaddeuskkr** [reply]
Nevermind, the PR I created was merged into the `develop` branch. You will either have to wait for the next release or use the `develop` branch in your app's package.json.
